docker/config/local_dev.env

```diff
@@ -88,5 +88,6 @@ RABBITMQ_PASSWORD=rabbitpwd
 RABBITMQ_VIRTUAL_HOST=rabbitvhost
 RAVEN_DSN=
 SECRET_KEY=secretkey
+STATSD_HOST=statsd
```

Comment:
`statsd` is the name of the statsd container and thus the hostname we need to use.